Traceback (most recent call last):
  File "minuit-min3.py", line 20, in ?
    compare(fit_result.values["x"], -3/2, precision=1e-6)
  File "/home/samuel/dev/pyhep/py/hep/test.py", line 59, in compare
    raise TestFailure, \
hep.test.TestFailure: expected -1.5; got 0.0
